brownie/test/plugin.py

```diff
@@ -104,6 +107,7 @@ def pytest_configure(config):
                 {"phases": {"explicit": True, "generate": True, "target": True}}, "brownie-failfast"
             )
 
+        active_project = project.get_loaded_projects()[-1]
```

Comment:
This should probably target the first list item instead of the last, in case the user loaded a 2nd project in a conftest file.

Reply:
```suggestion
        active_project = project.get_loaded_projects()[0]
```